src/elfpy/pricing_models/yieldspace.py

```diff
@@ -357,7 +357,7 @@ def calc_in_given_out(
         # We precompute the YieldSpace constant k using the current reserves and
         # share price:
         #
-        # k = (c / μ) * (μ * z)**(1 - t) + (2y + cz)**(1 - t)
+        # k = (c / μ) * (μ * z)**(1 - τ) + (2y + cz)**(1 - τ)
```

Comment:
Let's pick $\tau$ or `tau` in this PR. I don't really care, but would be good to set the record straight. Let's do the same for picking between $\mu$ and `mu`.

Reply:
I went with `mu` and `tau`.  I cleaned a LOT of this up:
`τ`, `t`, `T`, `tau` -> `tau`.
`T`, `t_stretch` -> `t_stretch`
`t` or `t_d` is now only used for normalized time
`μ`, `u` -> `mu`